### PR TITLE
Add Value Tooltips

### PR DESCRIPTION
**Pull request by istarkov**
## Description

ref #1798

- [x] - Text button groups tooltips should be resetable see https://discord.com/channels/955905230107738152/1128272360630399016/1128800552743084263 as no other way to reset it (except alt button)

Value Tooltips for toggle groups

- [x] - Also fixed tooltip hovering issues on toggle groups see grace area
image

If group has no label we show reset on button

image

---

image

I also added tooltips on `...` Buttons, it shows all current values of dialog underline

image

## Steps for reproduction

Toggle groups tooltips changed:
image

image
image
image
image

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/builder/features/style-panel/sections/backgrounds/background-content.tsx [hunk 1]
```diff
@@ -276,23 +276,35 @@ export const BackgroundContent = (props: BackgroundContentProps) => {
                   items={[
                     {
                       child: <CrossSmallIcon />,
-                      label: "background-repeat: no-repeat",
+                      title: "BackgroundRepeat",
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/sections/backgrounds/background-content.tsx:279]: Title should be humanized, no? e.g. "Background Repeat"

**Comment by kof** [comment]
- [ ] Where is the reset button? did we decide to not include it?
- [ ] I can't hover the tooltip now, it disappears as soon as I try

**Comment by kof** [comment]
- [ ] not sure if it happened in this pr but here layout broke
Screenshot 2023-07-13 at 12 52 59

**Comment by istarkov** [comment]
> * [ ]  not sure if it happened in this pr but here layout broke
> 
> Screenshot 2023-07-13 at 12 52 59

Not in this, exists on production too, will fix anyway

**Review by kof — APPROVED** [review]

**Pull request merged**